aalto/jupyterhub/nbgrader-jupyterlab.rst

```diff
@@ -0,0 +1,83 @@
+Doing Assignments in JupyterLab
+===============================
+
+.. note::
+
+    This is a tutorial for courses that distribute and collect
+    their assignments on `JupyterHub <https://jupyter.cs.aalto.fi>`__.
+    Feel free to use JupyterHub for your other courses as well, but
+    note that you won't be able to collect and submit your
+    assignments this way.
+
+Go to https://jupyter.cs.aalto.fi and sign in with your account to access JupyterHub.
+You can find more general information about it in :doc:`../jupyterhub`.
+Select your course and start a session. Your will see that you are under the ``notebooks/`` folder,
+which is your home directory. Make sure you put everything important here, files saved
+elsewhere will get deleted when your session ends.
+
+.. figure:: img/home.png
+   :align: center
+|
+You can see currently available assignments by going to "Nbgrader->Assignment List"
```

Comment:
If this is supposed to go in the figure caption (which I think it is, right), this is how it would be done.  I can fix the rest if this I correct:
```suggestion
.. figure:: img/home.png
   :align: center

   You can see currently available assignments by going to "Nbgrader->Assignment List"
```

Reply:
In general, I didn't intend the text to be captions (they are not optional in going through the tutorial), and most of them refer to the images below them rather than above. But I can change them if you think it makes more sense/looks better.